pilot/pkg/networking/util/util.go

```diff
@@ -23,8 +23,12 @@ import (
 	xdsapi "github.com/envoyproxy/go-control-plane/envoy/api/v2"
 	"github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
 	"github.com/envoyproxy/go-control-plane/pkg/util"
+	"github.com/gogo/protobuf/jsonpb"
 	"github.com/gogo/protobuf/proto"
 	"github.com/gogo/protobuf/types"
+	"k8s.io/apimachinery/pkg/util/json"
```

Comment:
doesn't encoding/json suffice?

Reply:
i think this is a lean & mean version - i got it from @costinm